core/services/ocr2/delegate.go

```diff
@@ -17,11 +17,12 @@ import (
 	"github.com/smartcontractkit/libocr/commontypes"
 	libocr2 "github.com/smartcontractkit/libocr/offchainreporting2plus"
 	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
-	ocr2keepers20 "github.com/smartcontractkit/ocr2keepers/pkg"
-	ocr2keepersconfig "github.com/smartcontractkit/ocr2keepers/pkg/config"
-	ocr2keepers20coordinator "github.com/smartcontractkit/ocr2keepers/pkg/coordinator"
-	ocr2keepers20polling "github.com/smartcontractkit/ocr2keepers/pkg/observer/polling"
-	ocr2keepers20runner "github.com/smartcontractkit/ocr2keepers/pkg/runner"
+	ocr2keepers20 "github.com/smartcontractkit/ocr2keepers/pkg/v2"
+	ocr2keepers20config "github.com/smartcontractkit/ocr2keepers/pkg/v2/config"
+	ocr2keepers20coordinator "github.com/smartcontractkit/ocr2keepers/pkg/v2/coordinator"
+	ocr2keepers20polling "github.com/smartcontractkit/ocr2keepers/pkg/v2/observer/polling"
+	ocr2keepers20runner "github.com/smartcontractkit/ocr2keepers/pkg/v2/runner"
+	ocr2keepers21config "github.com/smartcontractkit/ocr2keepers/pkg/v3/config"
```

Comment:
🏄 Same for all these? But totally fine if you'd prefer this way instead 👍

Reply:
discussed offline, prefer to keep it verbose as this is shared with other products (ocr2vrf etc)